.github/workflows/assembly.yml

```diff
@@ -41,6 +41,9 @@ jobs:
     - name: Build with Maven
       run: |
         mvn -B clean install -DskipTests -Dall -T2 --file pom.xml
+    - name: Javadoc build
+      run: |
+        mvn -B -DskipTests -Dfmt.skip=true javadoc:aggregate
```

Comment:
Do you need to specify the pom.xml file here?  I am not sure, it just does not look consistent.

Reply:
The default `pom.xml` is assumed. GeoServer workflow only uses `-f` because it wants to build in specific directories.

Let me check the output to see if it worked...